bin/check-stale-results.rb

```diff
@@ -60,22 +61,32 @@ def humanize(secs)
     end.compact.reverse.join(' ')
   end
 
-  def api_request(method, path)
+  def get_uri(path)
+    protocol = (settings['api'].key?('protocol') ? settings['api']['protocol'] : 'http')
+    uri = URI(protocol + '://' + settings['api']['host'] + ':' + settings['api']['port'].to_s + path)
```

Comment:
It's better to use string interpolation over string concatenation for the following reasons:
- string concatenation requires creating 3 strings for combining 2 strings therefore is more exspensive
- it automatically calls `.to_s` for you so there is no need to do that
- it is more common practice in the community
- more idiomatic
- less typing
- better use of chars on a line
```ruby
URI("#{protocol}://#{settings['api']['host']}:#{settings['api']['port']}#{path}")
```

Also you don't need to assign it to a var if you are just returning it you you can remove the `url` below and remove the assignment.

Reply:
Nice, thanks, changes made.
I've spent most of my career writing java so Ruby is "new" and I don't know it that well.

Reply:
I view open source as one big playground of learning. This is why code review happens, we all learn from things and improve the code. I spent most of my career with bash and python up until a few years ago, so I definitely understand that you often bring familiar patterns with you. I use to write really great python in my ruby, now I write great ruby in my python. Each language has its own little quirks and contributing to open source is a great way to learn them as someone will generally point out something you did no know.